README.md

```diff
@@ -38,6 +38,15 @@ When `prebuild-install` is run via an `npm` script, options
 `--build-from-source`, `--debug` and `--download`, may be passed through via
 arguments given to the `npm` command.
 
+### Custom binaries
+The end user can override binary download location through environment variables in their .npmrc file.  
+The variable needs to meet the mask `% your package name %_binary_path`.  
+E.g.
+```
+leveldown_binary_site=http://overriden-host.com/overriden-path
+```
+Note that the package version subpath and file name will still be appended.
```

Comment:
Can you give an example for the resulting url, for others?

Reply:
sure, updated readme